### DISCUSSION TITLE
Proxmox 8.2 ISO file with arm64

### DISCUSSION THREAD

**Discussion by SQuent** [jiangcuo/Proxmox-Port#108] [Q&A]
Hello,
Is it possible to have the iso file for Proxmox 8.2 available in repository ? Or how can I build it by myself ?

I want to test it with the new way of automated install from the ISO with answer file on Raspberry Pi 4.
Thank's everybody for help,
Thank you @jiangcuo for the work!

**Comment by jiangcuo** [comment]
New iso uploaded.
This is pveiso build tool.
https://github.com/jiangcuo/pve-iso-builder
